functions/zoom-downloader.py

```diff
@@ -494,7 +495,7 @@ def stream(self):
 
     @property
     def file_extension(self):
-        return self.zoom_filename.split(".")[-1]
+        return splitext(self.zoom_filename)[-1]
```

Comment:
This is creating a filename with two "."
Like so:
"577132153/1585344688/001-shared_screen_with_speaker_view..mp4"

Reply:
You can just remove the period in the s3_filename function

Reply:
oh wow good catch. i did not think it worked that way. i shouldn't be making unrelated changes like this anyway.